MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -807,12 +807,29 @@ private async Task<string> ExtractImageInternal(string inputPath, string contain
             int? threads,
             int? qualityScale,
             ProcessPriorityClass? priority,
+            bool enableKeyFrameOnlyExtraction,
             EncodingHelper encodingHelper,
             CancellationToken cancellationToken)
         {
             var options = allowHwAccel ? _configurationManager.GetEncodingOptions() : new EncodingOptions();
             threads ??= _threads;
 
+            if (enableKeyFrameOnlyExtraction)
```

Comment:
```suggestion
            if (allowHwAccel && enableKeyFrameOnlyExtraction)
```

Then the conditional can be simplified a little